File: hellocharts/src/main/java/net/raquezha/lecho/hellocharts/listener/ComboLineColumnChartOnValueSelectListener.java
package net.raquezha.lecho.hellocharts.listener;


import net.raquezha.lecho.hellocharts.model.PointValue;
import net.raquezha.lecho.hellocharts.model.SubcolumnValue;

public interface ComboLineColumnChartOnValueSelectListener extends OnValueDeselectListener {

    void onColumnValueSelected(int columnIndex, int subColumnIndex, SubcolumnValue value);

    void onPointValueSelected(int lineIndex, int pointIndex, PointValue value);

}

File: hellocharts/src/main/java/lecho/lib/hellocharts/listener/ComboLineColumnChartOnValueSelectListener.kt
package lecho.lib.hellocharts.listener

import lecho.lib.hellocharts.model.PointValue
import lecho.lib.hellocharts.model.SubcolumnValue

interface ComboLineColumnChartOnValueSelectListener : OnValueDeselectListener {
    fun onColumnValueSelected(columnIndex: Int, subColumnIndex: Int, value: SubcolumnValue)
    fun onPointValueSelected(lineIndex: Int, pointIndex: Int, value: PointValue)
}